trollflow2/launcher.py

```diff
@@ -217,6 +217,29 @@ def expand(yml):
     return yml
 
 
+def get_dask_client(config):
+    """Create Dask client if configured."""
+    client = None
+
+    try:
+        client_class = config["dask_distributed"]["class"]
+        client = client_class(**config["dask_distributed"]["settings"])
```

Comment:
How about leaving this empty if no settings are provided ? So the default behaviour will be used (the scheduler and worker is created automatically, good for testing)

Reply:
When this setting isn't in the config we'll use the default scheduler: no local cluster, no clients. That is, the current behaviour, which is often the most efficient in any case.

Reply:
ok, I mean that if the scheduler address isn't specified, the distributed scheduler would still be used but with default parameters. If the `dask_distributed` setting, or `class` isn't defined, we fallback to the default scheduler.

Reply:
Ah, ok. That's what happens here. If I define the config like this:
```yaml
dask_distributed:
  class: !!python/name:dask.distributed.Client
  settings: {}
```
a local cluster is created with default parameters. In the settings the user can also define any keyword argument the `Client` class supports for creating a local cluster.

Reply:
Actually, I'll make the `settings` optional to make the config simpler.